community/faas/README.md

```diff
@@ -0,0 +1,127 @@
+# Creating Projects Through Deployment Manager
```

Comment:
This seems to be related to the project factory template, which is not in this folder or PR. Can we remove unrelated information?

Reply:
fixed